bids2openminds/mapping.py

```diff
@@ -0,0 +1,95 @@
+experimental_approach={

```

Comment:
either use plural or singular, but do not mix this for the different dictionaries

Reply:
I think we should also follow the constant convention in python here and identify that this is a map:

- MAP_2_EXPERIMENTAL_APPROACHES or
- MAP_TO_EXPERIMENTAL_APPROACHES